react.js

```diff
@@ -1,67 +1,106 @@
+
 // ASSESSMENT 4: REACT ASSESSMENT
 // Coding practical questions
 
 // NOTE: In this assessment you will NOT be running a react file structure
 
 // 1. Write a React component that prints "I am a component!" Be sure to include all necessary imports, exports, etc.
 
-
-
+// <script type="text/babel">
+//       class HelloReact extends React.Component{
+//         render(){
+//           return(
+//             <h1>I am a Component!</h1>
+//           )
+//         }
+//       }
+//       const domContainer = document.querySelector('#react-container');
+//       ReactDOM.render(<HelloReact />, domContainer);
+//     </script>
 
 
 // 2. Refactor this vanilla javascript loop to a map function. The output should remain the same.
 
-var names = ["Ford Prefect", "Arthur Dent", "Trillian", "Zaphod", "Vogon", "Marvin, the Paranoid Android"]
+// var  ames = ["Ford Prefect", "Arthur Dent", "Trillian", "Zaphod", "Vogon", "Marvin, the Paranoid Android"]
 
-for(let i=0; i<names.length; i++){
-  console.log(`${names[i]} is ${names[i].length} characters long.`)
-}
+// for(let i=0; i<names.length; i++){
+//   console.log(`${names[i]} is ${names[i].length} characters long.`)
+// } 
 
+// names.map((value, index) => {
+// console.log(`${names[index]} is ${names[index].length} characters long.`)
+
```

Comment:
Looks good, but since you are passing value as an argument to map  you should use ${value} instead of ${names[index]}